omnibus/files/private-chef-cookbooks/private-chef/recipes/add_ons_wrapper.rb

```diff
@@ -48,14 +48,16 @@ def report
 
   require 'mixlib/install'
 
+  platform_info = Mixlib::Install.detect_platform
```

Comment:
I should clarify.  There is nothing wrong with the way you did it.  But, since you need to create an instance any to call `artifact_info` I suggest using the instance method.